iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift

```diff
@@ -7,15 +7,50 @@
 //
 
 import SwiftUI
+import shared
 
 struct AddMealView: View {
-    var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
-    }
+
+	@ObservedObject private(set) var viewModel: ViewModel
+	
+	var body: some View {
+		//TODO: Pass the actual values when UI is complete
+		let sdk = MealApi()
+		Button("Add a meal") {
+			let meal = Meal(id: "\(sdk.getUUID())",
```

Comment:
Could we switch this out to:
`let meal = Meal(id: "\(viewModel.sdk.getUUID())",`

and get rid of the `let sdk = MealApi()` line please

Reply:
Done